help/include/trying-out-zulip.md

```diff
@@ -3,17 +3,44 @@ topic-based threading [combines the best of email and
 chat](/why-zulip/) to make [remote work](https://zulip.com/for/business/) productive
 and delightful.
 
-We also highly recommend trying Zulip for yourself! You can:
+It's also easy to try out Zulip for yourself.
 
-* [Check out the Zulip app](https://zulip.com/try-zulip/) by viewing the Zulip
-  development community, where hundreds of participants collaborate to improve
-  Zulip. Many parts of the community are open for [public
-  access](/help/public-access-option), so you can start exploring without
-  creating an account.
+{start_tabs}
 
-* [Create a Zulip Cloud organization](https://zulip.com/new/) for free in just a
-  few minutes.
+1. **Check out the Zulip app**. You can:
 
-* [See how Zulip is being used](https://zulip.com/communities/) in open
-  organizations that have opted in to be listed in the [Zulip communities
-  directory](/help/communities-directory).
+    - [View the Zulip development community](https://zulip.com/try-zulip/),
+      where hundreds of participants collaborate to improve Zulip. Many parts of
+      the community are open for [public access](/help/public-access-option), so
+      you can start exploring without creating an account.
+
+    - [See how Zulip is being used](https://zulip.com/communities/) in open
+      organizations that have opted in to be listed in the [Zulip communities
+      directory](/help/communities-directory).
+
+    - [Create a Zulip Cloud organization](https://zulip.com/new/) for free in just a
+      few minutes.
+
+1. [Invite your teammates](/help/invite-users-to-join) to explore Zulip with you
+  and share their unique perspectives.
```

Comment:
I feel like we want something more along the lines of "explore Zulip with you and discuss how Zulip might help their work."

Reply:
Trying a different variant (screenshot updated). What do you think?